### PR TITLE
Add impact calculator

### PR DESCRIPTION
**Pull request by har777**

### CODE REVIEW

File: components/BuyingGuide.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <div className="flex flex-col space-y-4 text-pine-900 mt-3 mb-5 mx-4 text-sm">
+        <div>
+          As Glo is in beta you can only get Glo by exchanging it with USDC
+          using an app called Uniswap. This process consists of 12 steps:
+        </div>
+
+        <ol className="list-decimal mx-4">
+          <li>Create an account with Coinbase</li>
+          <li>Buy {glo + 1} USDC on Coinbase via wire transfer</li>
+          <li>Send {glo + 1} USDC to: 0x...ef81 🔗 on Polygon</li>
```
Comment (gglucass) [inline components/BuyingGuide.tsx:30]: wallet address should be dynamically populated :p

Comment (har777) [reply components/BuyingGuide.tsx]: fixed

File: components/BuyingGuide.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          <li>Go to app.uniswap.com</li>
+          <li>Click connect and choose WalletConnect</li>
+          <li>Switch to “QR Code” mode</li>
+          <li>Click on “Copy to clipboard”</li>
+          <li>On this app’s main page select “Scan”</li>
+          <li>Click WalletConnect to paste and connect</li>
+          <li>Visit the USDC {"<>"} Glo page on Uniswap 👇</li>
+          <li>Input {glo} USDC and click “Swap”</li>
+          <li>Complete the Uniswap steps to buy 1000 Glo!</li>
```
[the remark this answers is not in the document]
Comment (har777) [reply components/BuyingGuide.tsx]: missed this. fixed.

File: components/BuyingGuide.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      </div>
+      <div className="flex flex-col space-y-4 text-pine-900 mt-3 mb-5 mx-4 text-sm">
+        <div>
+          As Glo is in beta you can only get Glo by exchanging it with USDC
+          using an app called Uniswap. This process consists of 12 steps:
+        </div>
+
+        <ol className="list-decimal mx-4">
+          <li>Create an account with Coinbase</li>
+          <li>Buy {glo + 1} USDC on Coinbase via wire transfer</li>
```
[the remark this answers is not in the document]
Comment (har777) [reply components/BuyingGuide.tsx]: updated.

**Review by gglucass — APPROVED** [review]

**Comment by gglucass** [comment]
Things still to be done from my perspective:
1. Deeplink into uniswap pair
2. Copy paste the wallet address on click 
3. I need to make some copy changes

**Pull request merged**